src/test/ui/consts/single_variant_match_ice.rs

```diff
@@ -15,7 +15,7 @@ impl Foo {
         use self::Foo::*;
 
         match *self {
-            Prob => 0x1, //~ ERROR `if`, `match`, `&&` and `||` are not stable in const fn
+            Prob => 0x1, //~ ERROR `if`, `match`, `while`, `for`, `loop`, `&&` and `||` are not stable in const fn
```

Comment:
move expectation comment to the next line and use the `//~^` notation get around tidy